challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -15319,6 +15319,1381 @@
+            "assert(typeof RomanDecode(\"MCMXC\") == 'number','<code>RomanDecode(\"MCMXC\")</code> should return a number.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MCMXC\")</code> should return <code>1990</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MCMXC\"),1990,'<code>RomanDecode(\"MCMXC\")</code> should return <code>1990</code>.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MDCLXVI\")</code> should return <code>1666</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MDCLXVI\"),1666,'<code>RomanDecode(\"MDCLXVI\")</code> should return <code>1666</code>.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MMVIII\")</code> should return <code>2008</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MMVIII\"),2008,'<code>RomanDecode(\"MMVIII\")</code> should return <code>2008</code>.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MDCCCXXXIX\")</code> should return <code>1839</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MDCCCXXXIX\"),1839,'<code>RomanDecode(\"MDCCCXXXIX\")</code> should return <code>1839</code>.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MMMMMMDCCXLV\")</code> should return <code>6745</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MMMMMMDCCXLV\"),6745,'<code>RomanDecode(\"MMMMMMDCCXLV\")</code> should return <code>6745</code>.');"
+        },
+        {
+          "text":
+            "'<code>RomanDecode(\"MMMMDCCCXXI\")</code> should return <code>4821</code>.'",
+          "testString":
+            "assert.equal(RomanDecode(\"MMMMDCCCXXI\"),4821,'<code>RomanDecode(\"MMMMDCCCXXI\")</code> should return <code>4821</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7fba",
+      "challengeType": 5,
+      "releasedOn": "September 6, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function RomanDecode (str) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Roots of a function",
+      "description": [
+        "Create a function that finds and returns the roots of a given function as an array.",
+        "In this task, you need to use brute force and put values in the given function to check if it is a root.",
+        "The function should take an array and 3 numbers as parameters. The array represents the function. The $ i^{th} $ element is the coefficient of $ x^i $. The numbers are lower bound, upper boand and step values. You need to start from the lower bound, increament by step value until the upper bound is reached. The values obtained should be tested if they are the roots of the given function."
```

Comment:
`"The function should take an array and 3 numbers as parameters. The array represents the function. The $ i^{th} $ element is the coefficient of $ x^i $. The numbers are the lower bound, upper bound, and step values, respectively. You need to start from the lower bound and increment by the step value until the upper bound is reached. The values obtained should be tested to see if they are the roots of the given function."`